app/assets/stylesheets/protocols/protocol.scss

```diff
@@ -0,0 +1,32 @@
+// scss-lint:disable SelectorDepth
+// scss-lint:disable NestingDepth
+
+@import "constants";
+@import "mixins";
+
+.protocol-info {
+
+  .well {
```

Comment:
I'm not sure why this styles cannot be used from the `protocols/index.scss'. They seem pretty similar, at least this `.well` is exactly the same. Could this be somehow DRYed up so it does not appear on 2 different places?

Reply:
DRYed